src/Bicep.Core.UnitTests/Utils/TextByteArray.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Bicep.Core.UnitTests.Utils;
+
+/// <summary>
+/// A byte array for UTF-8 text
+/// </summary>
+public class TextByteArray
+{
+    private readonly ImmutableArray<byte> _bytes;
```

Comment:
nit: We've been avoiding use `_` as a prefix for field names.

Reply:
Ah, yeah.  I pulled this from my earlier PR which was before this was pointed out.  Fixing.

I tried to enforce in .editorconfig but couldn't get it to work.